### PR TITLE
Bump  WC versions for 5.7.0 release

### PR DESCRIPTION
**Pull request by cesarcosta99**
Bump WC/WP versions according to the L-2 policy (see https://protonpower.wordpress.com/ and https://wordpress.org/news/category/releases/)

Current WC version is 7.5, but since this isn't bumped in a while I've changed it to 7.2. We'll be able to catch up the L2 policy in a couple of releases.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `5844` or branch name `dev/bump-versions-5-7-0` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=dev/bump-versions-5-7-0) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: ba6387b41241cd940e58822e71ee59cecf509c61
- Build time: 2023-03-29 18:46:31 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Comment by jbordonado** [comment]
We used to bump 2 by 2 to catch-up on L-2 version before.
Any reason why we wouldn't want to bump by 2 and catch-up already this time?

**Comment by cesarcosta99** [comment]
> We used to bump 2 by 2 to catch-up on L-2 version before. Any reason why we wouldn't want to bump by 2 and catch-up already this time?

@jbordonado  I actually didn't recall that, thanks for raising it. Updated minimum version to 7.3.0.

**Comment by jbordonado** [comment]
Can we get it reviewed and merged before the release? 
Looks fine to me.

Previous PR for this, for reference: #5336

File: changelog/dev-bump-versions-5-7-0 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Bump minimum required version of WooCommerce to 7.2.3.
```
Comment (ricardo) [inline changelog/dev-bump-versions-5-7-0]: Should this be 7.3.0?

File: woocommerce-payments.php [hunk 2]
```diff
@@ -8,8 +8,8 @@
  * Woo: 5278104:bf3cf30871604e15eec560c962593c1f
  * Text Domain: woocommerce-payments
  * Domain Path: /languages
- * WC requires at least: 7.1
- * WC tested up to: 7.3.0
+ * WC requires at least: 7.3
+ * WC tested up to: 7.5.0
```
Comment (ricardo) [inline woocommerce-payments.php]: The latest WC release right now is [7.5.1](https://github.com/woocommerce/woocommerce/releases/tag/7.5.1).

**Review by ricardo — APPROVED** [review]
LGTM. Please check the comments before merging.

File: changelog/dev-bump-versions-5-7-0 (hunk 1 above)
Comment (cesarcosta99) [reply changelog/dev-bump-versions-5-7-0]: Yes, fixed in 15ed5cd.

File: woocommerce-payments.php (hunk 2 above)
Comment (cesarcosta99) [reply woocommerce-payments.php]: Updated in d7d98c3. Thanks for checking.

**Comment by cesarcosta99** [comment]
PHP tests are passing locally, so I'm going to go ahead and merge this. I guess the 2 new failures is a CI-kind of issue.

**Pull request merged**